.github/workflows/validate-component-metadata-schema.yml

```
@@ -0,0 +1,45 @@
+name: validate-metadata-schema
+env:
+  PYTHON_VERSION: 3.11
+  COMPONENT_SCRIPT_PATH: $GITHUB_WORKSPACE/.github/resources/scripts/get_components_for_validation.sh
+  VALIDATION_SCRIPT_PATH: $GITHUB_WORKSPACE/scripts/validate_metadata/validate_metadata.py
+on:
+  pull_request:
+    paths:
+      - 'component/pipelines/**'
+      - 'scripts/**'
+
+jobs:
+  validate-component-metadata-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install Test dependencies
+        run: |
+          pip install -r scripts/validate_metadata/requirements.txt
+
+      - name: Retrieve new components
+        id: get-new-components
+        run: ${{  env.COMPONENT_SCRIPT_PATH  }} "${{ github.event.pull_request.base.sha }}" "${{ github.event.pull_request.head.sha }}" validate_metadata
+
+      - name: Validate new components
+        if: ${{ steps.get-new-components.outputs.new_components_list != '' }}
+        run: |
+          NEW_COMPONENTS_ARRAY="${{ steps.get-new-components.outputs.new_components_list }}"
+
+          # 2. Set IFS to a comma, so that the shell will split the string by commas.
```

there is either a missing step `1. .....` or you should remove "2. "
```suggestion
          # Set IFS to a comma, so that the shell will split the string by commas.
```